src/plugins/build.ts

```diff
@@ -4,9 +4,8 @@ import { version } from "../../package.json";
 import { networkEthId } from "~/types/lib";
 
 export const _ETHER_NETWORK_ID_DICTIONARY: networkEthId[] = [
-  { name: "rinkeby", id: 4 },
-  { name: "ropsten", id: 3 },
-  { name: "mainnet", id: 1 },
```

Comment:
@antomor this change could imply that you need to change the `APP_CURRENT_NETWORK=testnet` instead of `APP_CURRENT_NETWORK=rinkeby` and `ZK_NETWORK=testnet` instead of `ZK_NETWORK=rinkeby-beta` too on the env file.

Reply:
Currently, the dapp has been already built with those settings